src/krasa/grepconsole/plugin/DefaultState.java

```diff
@@ -52,42 +53,37 @@ public static void resetToDefault(Profile profile) {
 
 	}
 
+	private static final ColorKey FATAL_BACKGROUND = ColorKey.createColorKey("GrepConsole.fatal.background", new JBColor(
+			JBColor.RED, new Color(0, 0, 0, 255)
+	));
+	private static final ColorKey ERROR_BACKGROUND = ColorKey.createColorKey("GrepConsole.error.background", new JBColor(
+			JBColor.ORANGE, new Color(55, 0, 0, 200)
+	));
+	private static final ColorKey WARN_BACKGROUND = ColorKey.createColorKey("GrepConsole.warn.background", new JBColor(
+			JBColor.YELLOW, new Color(26, 0, 55, 200)
+	));
+	private static final ColorKey DEBUG_FOREGROUND = ColorKey.createColorKey("GrepConsole.debug.foreground", new JBColor(
+			JBColor.GRAY, JBColor.GRAY
+	));
+	private static final ColorKey TRACE_FOREGROUND = ColorKey.createColorKey("GrepConsole.trace.foreground", new JBColor(
+			JBColor.LIGHT_GRAY, JBColor.BLACK
+	));
+
 	public static List<GrepExpressionItem> createDefaultItems(boolean dark, boolean enabled) {
```

Comment:
Wasn't exactly sure what to do with the `dark` key.

Right now the settings menu's light and dark theme color key (for default values) show the same color based on the current editor scheme. So if the current scheme is dark, both the light and dark color check boxes show the same color. However, because the underlying color is determined by the editor scheme, when the user changes themes, the highlighting is correct.

Didn't know what to do for that, because you said something along the lines of eventually removing the light & dark settings.

Reply:
Yep, I removed it.